docs/src/main/asciidoc/mp/persistence.adoc

```diff
@@ -327,7 +327,7 @@ the `test` data source's associated connection pool or vendor-specific
 
 [source,properties]
 ----
-javax.sql.DataSource.test.foo.bar=baz # <1> <2> <3>
+javax.sql.DataSource.test.foo.bar=baz# <1> <2> <3>
```

Comment:
I think you need to remove spaces between callouts too
`javax.sql.DataSource.test.foo.bar=baz# <1><2><3>`